package.json

```
@@ -26,13 +26,13 @@
         "danger": "^10.1.1",
         "dtslint": "latest",
         "prettier": "^2.0.2",
-        "@definitelytyped/dtslint-runner": "latest"
+        "@definitelytyped/dtslint-runner": "latest",
+        "typescript": "next"
```

To make the diff cleaner, I suggest putting `typescript@next` before `@definitelytyped/dtslint‑runner`, like in <https://github.com/DefinitelyTyped/DefinitelyTyped/pull/45226>:
```suggestion
        "typescript": "next",
        "@definitelytyped/dtslint-runner": "latest"
```

---

And I suggest using `npx sort-package-json` that will produce this:
```json
    "devDependencies": {
        "@definitelytyped/dtslint-runner": "latest",
        "danger": "^10.1.1",
        "dtslint": "latest",
        "prettier": "^2.0.2",
        "typescript": "next"
    }
```
The reason is that `package.json` will be re-ordered anyway in case if a package will be added/removed using `npm` cli.